Multi-Module Plugin Configuration

Is the documentation for a multi-module plugin configuration still accurate? When I try creating a project with that configuration I get a build error. I created a simple project to replicate the issue: https://github.com/austinarbor/pitest-example

When I run any gradle command I get a build error:

```shell
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/austinarbor/git/pitest-example/build.gradle' line: 21

* What went wrong:
A problem occurred evaluating root project 'pitest-example'.
> Could not find method pitest() for arguments [build_7tx83dncip5nr5k5thb517upf$_run_closure1$_closure3@21cefca8] on project ':submodule-1' of type org.gradle.api.Project.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/build.gradle
+++ b/build.gradle
@@ -17,6 +17,7 @@ group 'dev.aga'
 version '1.0-SNAPSHOT'
 
 subprojects {
+    apply plugin: 'java'
     apply plugin: 'info.solidsoft.pitest'
     pitest {
         junit5PluginVersion = '1.0.0'

--- a/build.gradle
+++ b/build.gradle
@@ -7,8 +7,8 @@ buildscript {
     }
 }
 
-plugins {
-    id 'java'
+repositories {
+    mavenCentral()
 }
 
 apply plugin: 'info.solidsoft.pitest.aggregator'
@@ -22,6 +22,7 @@ subprojects {
     pitest {
         junit5PluginVersion = '1.0.0'
         threads = Runtime.getRuntime().availableProcessors()
+        outputFormats = ["XML"]	//required for aggregated report
         exportLineCoverage = true
         timestampedReports = false
         targetClasses = ['dev.aga.*']
@@ -29,6 +30,3 @@ subprojects {
     }
 }
 
-test {
-    useJUnitPlatform()
-}

--- a/build.gradle
+++ b/build.gradle
@@ -1,10 +1,5 @@
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.9.0'
-    }
+plugins {
+  id "info.solidsoft.pitest" version "1.9.0" apply false
 }
 
 repositories {

